examples/tools/go_db.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+import pyscf
+from pyscf.tools.go_db import get_xyz
```

Comment:
I've changed `go_db.py` to `get_xyz.py`. Is it possible to remove this `import` statement entirely? I'd prefer to minimize the number of imports in the example file, and to minimize the number of total lines. Users can choose to add this line if they want to simplify the code later in the file, but ideally an "example" file would be supremely simple, like this one: https://github.com/pyscf/pyscf/blob/master/examples/scf/00-simple_hf.py

Reply:
I think this change will make it confusing for people, especially because the module and the function are called the same name. If we would need to remove the import statement then we need to put the function in the same script.
Is that something that you would like? If yes, it would contradict your other statement that we need to minimize number of lines in total. 

The example at the moment is more compact in terms of number of lines when compared to having the function in the script. Let me know how you would like to tackle this.